ivy/array/general.py

```diff
@@ -99,7 +103,20 @@ def unstack(self: ivy.Array, axis: int, keepdims: bool = False) -> ivy.Array:
         -------
         ret
             List of arrays, unpacked along specified dimensions.
+        
+        Examples
+        --------
+        >>> x = ivy.array([[1, 2], [3, 4]])
+        >>> y = x.unstack(axis=0)
+        >>> print(y)
+        [ivy.array([1, 3]), ivy.array([2, 4])]
```

Comment:
I think the correct output here should be:
`
[ivy.array([1,2]),ivy.array([3,4])]
`